scripts/clusterCreation/getClusterAdminIDforDeployment.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
I don't think this file is needed anymore. Might have been from an old branch. There is similar logic in main here https://github.com/retaildevcrews/ngsa-asb/blob/main/scripts/clusterCreation/1-CheckPrerequisites.sh

Reply:
Actually, it referenced from file scripts/clusterCreation/2-CreateHub.sh (line 8)
But thanks for pointing that out

Reply:
Ok, Alfredo just confirmed that it is a typo and it should be referencing to scripts/clusterCreation/1-CheckPrerequisites.sh  instead,